l3_dump.py

```diff
@@ -1,18 +1,27 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
+"""
+Python script to unpack L3 logging file into human-readable trace messages.
+L3: Lightweight Logging Library, Version 0.1
+Date 2023-12-24
+Copyright (c) 2023-2024
+"""
```

Comment:
Applied `pylint` to this file, and that's included in the Makefile, too.

Future changes have to conform otherwise builds will fail.